src/content/docs/mobile-monitoring/new-relic-mobile-ios/get-started/new-relic-ios-and-dt.mdx

```diff
@@ -15,28 +15,30 @@ Watch this short video (approx. 2:15 minutes) to learn how to:
 
 To use distributed tracing, you need at least XCFramework agent version 7.3.0. We recommend you use the most recent agent.
 
-<Callout variant="tip">
-Infinite Tracing is not yet available for mobile monitoring.
-</Callout>
-
-## How to set up distributed tracing [#setup]
+## How to set up distributed tracing [#set-up-dt]
```

Comment:
I thought we should't rename anchors.

Reply:
@urbiz-nr, good catch!! I switched it back!